lib/ansible_tower_client/common.rb

```diff
@@ -0,0 +1,26 @@
+module AnsibleTowerClient
+  module Common
```

Comment:
What do you think of making this a superclass of the other three called `Collection`?